lib/storage/file.js

```diff
@@ -796,6 +796,145 @@ File.prototype.getSignedUrl = function(options, callback) {
 };
 
 /**
+ * Get a signed policy document to allow user to upload data
+ * with a POST.
+ *
+ * *[Reference](http://goo.gl/JWJEkG).*
+ *
+ * @throws {Error} if an expiration timestamp from the past is given or 
+ *     option parameter does not respect
+ *
+ * @param {object} options - Configuration object.
+ * @param {object} options.expiration - Timestamp (seconds since epoch) 
+ *     when this policy will expire.
+ * @param {object[][]=} options.equals - Array of request parameters and 
+ *     their expected value (e.g. [["$<field>", "<value>"]]). Values are 
```

Comment:
The `$` is required. As a user I would love to get rid of it. However, for a user coming from the official [docs](http://goo.gl/JWJEkG) removing it might be counter-intuitive.